pymc3/smc/sample_smc.py

```diff
@@ -35,7 +35,7 @@ def sample_smc(
     n_steps=25,
     start=None,
     tune_steps=True,
-    p_acc_rate=0.99,
+    p_acc_rate=0.85,
```

Comment:
Just realized my simulation use `.99` still and it seems better than `.85` - how confidence are you with this @aloctavodia?

Reply:
Not sure at all. This is something to test and most likely change. Eyeballing 0.9 could be a good compromise. But for you chains/steps are cheaper,right? So maybe you can keep 0.99.